src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -924,23 +922,14 @@ h2.small-section-header > .anchor {
 	-moz-box-sizing: border-box !important;
 	box-sizing: border-box !important;
 	outline: none;
-	border: none;
-	border-radius: 1px;
-	margin-top: 5px;
-	padding: 10px 16px;
+	border: 1px solid;
+	border-radius: 2px;
+	padding: 5px 8px;
```

Comment:
What do you think of applying these border and padding rules to `.search-container > *`? That would allow de-duplicating some items that are separately assigned to help button, settings button, and so on.